addons/crm/wizard/crm_lead_to_opportunity_mass_views.xml

```diff
@@ -18,20 +18,20 @@
                 </group>
                 <label for="duplicated_lead_ids" string="Leads with existing duplicates (for information)" help="Leads that you selected that have duplicates. If the list is empty, it means that no duplicates were found" invisible="not deduplicate"/>
                 <group invisible="not deduplicate">
-                    <field name="duplicated_lead_ids" colspan="4" nolabel="1" readonly="1">
-                        <tree create="false" delete="false">
-                            <field name="create_date"/>
-                            <field name="name"/>
-                            <field name="type"/>
-                            <field name="contact_name"/>
-                            <field name="country_id" column_invisible="context.get('invisible_country', True)" options="{'no_open': True, 'no_create': True}"/>
-                            <field name="email_from"/>
-                            <field name="stage_id"/>
-                            <field name="user_id" widget="many2one_avatar_user"/>
-                            <field name="team_id"/>
-                        </tree>
-                    </field>
-                </group>
+                    <field name="duplicated_lead_ids" colspan="2" nolabel="1" readonly="1">
+                    <tree create="false" delete="false">
+                        <field name="create_date"/>
+                        <field name="name"/>
+                        <field name="type"/>
+                        <field name="contact_name"/>
+                        <field name="country_id" column_invisible="context.get('invisible_country', True)" options="{'no_open': True, 'no_create': True}"/>
+                        <field name="email_from"/>
+                        <field name="stage_id"/>
+                        <field name="user_id" widget="many2one_avatar_user"/>
+                        <field name="team_id"/>
+                    </tree>
+                </field>
+</group>
```

Comment:
Some indentation issue here :) when copy pasting this happens often alas. Do not forget to double check ;) Previous indentation was correct as the tree view is 'inside' the field since it is a tree view of those records.